samples/client/petstore/lua/test.lua

```diff
@@ -0,0 +1,12 @@
+local petstore_client = require "petstore.api.pet_api"
```

Comment:
This is an example rather than what I'd call a test: intentional?

Reply:
Yup, agreed it's more like an example. I'll submit a separate PR with integration tests using https://github.com/bluebird75/luaunit

Reply:
Use busted, not luaunit

Reply:
Sure, I'll also add auto-generated test files for API and models.

Ref: https://github.com/Olivine-Labs/busted